vllm/compilation/decorators.py

```diff
@@ -447,6 +447,7 @@ def patched_inline_call(self_):
                     InliningInstructionTranslator, "inline_call_", patched_inline_call
                 ),
                 torch._dynamo.config.patch(**dynamo_config_patches),
+                set_current_vllm_config(self.vllm_config),
```

Comment:
This is a huge change, we shouldn't be settling the vllm config globally here

Reply:
Removed now that the inductor config is passed in explicitly.